pkgs/by-name/de/devcontainer-cli/package.nix

```diff
@@ -0,0 +1,97 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, fetchYarnDeps
+, makeWrapper
+, nodejs
+, prefetch-yarn-deps
+, yarn
+, git
+, docker
+, nodePackages
+, testers
+, devcontainer-cli
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "devcontainer-cli";
+  version = "0.53.0";
+
+  src = fetchFromGitHub {
+    owner = "devcontainers";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-WslXoyE+l26kiKVbwAKAYG5KkFmhT7zUY5mLTq+yP/8=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = ./yarn.lock;
+    hash = "sha256-Ybqjx7FN2J0IzO/rdKh7X57Nw08dzGZtjKr3PXLZEcg=";
+  };
+
+  nativeBuildInputs  = [
+    makeWrapper
+    nodejs
+    prefetch-yarn-deps
+    yarn
+  ];
+
+  runtimeDependencies = [
+    git
+    docker
+  ];
+
+  configurePhase = ''
+    runHook preConfigure
+
+    export HOME=$(mktemp -d)
+    yarn config --offline set yarn-offline-mirror "$offlineCache"
+    fixup-yarn-lock yarn.lock
+    yarn --offline --frozen-lockfile --ignore-platform --ignore-scripts --no-progress --non-interactive install
+    patchShebangs node_modules
+
+    runHook postConfigure
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    yarn --offline definitions
+    yarn --offline compile-prod
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    yarn --offline --production install
+
+    rm -rf node_modules/node-pty
+    ln -s ${nodePackages.node-pty}/lib/node_modules/node-pty node_modules/node-pty
```

Comment:
What is the error when we don't do this?

Reply:
As far as I can remember, it builds just fine but fails during runtime due to missing `../build/{Release,Debug}/pty.node`. The reason was that it's missing executables that were supposed to be built with node-gyp. That is the reason why I ended up packaging this package standalone as well.